src/JavaBasic123.java

```diff
@@ -1,5 +1,58 @@
 public class JavaBasic123 {
     public static void main(String[] args) {
-        
+
+
+        String str = "This is very easy programme ";
+        int len = str.length();
+        String spchar = "This $300 is equivalent of £200 but not for @100";
+        spchar = spchar.replace("$", "");
+        spchar = spchar.replace("£", "");
+        spchar = spchar.replace("@", "");
+        String v = "The cost of iphone is £1,250 ";
+        String v1 = v.replaceAll("[0-9]", "");
+        int a = 450;
+        double a1 = a;
+        String b = "125";
+        int i = Integer.parseInt(b);
+        double d = 123.45;
+        String s = Double.toString(d);
+        float c = 1 / 3;
+        String k = "560.25";
+        double t = Double.parseDouble("560.25");
+        String price = "$250.00 ";
+        String word1 = "There ";
+        word1 = word1.substring(2, 3);
+        String word2 = "England";
+        word2 = word2.substring(3, 4);
+        String word3 = "orange";
+        word3 = word3.substring(2, 4);
+        System.out.print(" Removing letter  s  from the sentence :");
+        System.out.println(str.replaceAll("s", " "));//removing letter 's' from the sentence
+        System.out.println(" String Length is : " + len);//finding the length of the sentence
+        System.out.print(" Removing spaces from the sentence  :");
+        System.out.println(str.replaceAll("[^a-zA-Z0-9]", ""));//removing spaces from the string
+        System.out.print(" Removing all special characters from the sentence  :");
```

Comment:
use string concatenation to reduce number of lines e.g. `System.out.print(" Removing all special characters from the sentence  :" + spchar);` at all possible places